test/avocado/seleniumlib.py

```diff
@@ -126,8 +134,14 @@ def send_keys(self, element, text, clear=True):
             self.driver.execute_script('var ev = document.createEvent("Event"); ev.initEvent("change", true, false); arguments[0].dispatchEvent(ev);', element)
             self.driver.execute_script('var ev = document.createEvent("Event"); ev.initEvent("keydown", true, false); arguments[0].dispatchEvent(ev);', element)
 
-    def wait(self, method, text, baseelement, overridetry, fatal, cond):
+    def wait(self, method, text, baseelement, overridetry, fatal, cond, jscheck):
         """
+This function is only for internal purposes:
```

Comment:
How about:
This function is low level, tests should prefer to use the wait_* functions: